runtime/common/WrappedKernel.h

```diff
@@ -0,0 +1,345 @@
+/****************************************************************-*- C++ -*-****
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+#pragma once
+#include "cudaq/platform.h"
+#include "cudaq/utils/registry.h"
+#include <cstdint>
+#include <cstring>
+#include <string>
+#include <vector>
+
+/*! \file WrappedKernel.h
```

Comment:
Maybe `KernelWrapper.h` is more suitable name...

Reply:
Agreed. Update the filename.